.github/golangci-lint.config.yaml

```diff
@@ -32,9 +32,9 @@ linters:
     - ineffassign
     - misspell
     - unconvert
-    - deadcode
-    - varcheck
```

Comment:
These are deprecated, they've been surpassed with `unused`, but `unused` is a beast and one we should tackle in the future, but not now.